### DISCUSSION TITLE
Can't change owner for db files

### DISCUSSION THREAD

**Discussion by ghost** [Ylianst/MeshCentral#5957] [Issues]
Hi,
I'm trying to change the owner of these files to be on my user and it keeps getting back to root.
As a result the server can't start automatically due to permission issue.
Thanks.

**Comment by si458** [comment]
it depends how u are running it in the first place.
If u run `sudo node node_modules/meshcentral` then it will run as root
If you run as `node node_modules/meshcentral` then it will run as user
You can check also what your user is by running `whoami`

**Reply by si458** [reply]
Also you need to run `sudo setcap 'cap_net_bind_service=+ep' /usr/bin/node` to allow ur nodejs to run on ports below 1024

**Reply by ghost** [reply]
How can I reinstall the Mesh from scratch? 
I want to clean totally and reinstall.

**Reply by si458** [reply]
Just simply delete all folders `node_modules`, `meshcentral_*` and also delete the meshcentral.service file too

**Reply by ghost** [reply]
Thank you.
I asked so I can start again from scratch as I'm not able to solve the issues I mentioned before.
Hope this will helps and I will notify if it worked.

**Reply by ghost** [reply]
Hey just wanted to inform that I was able to fix by downgrading to 1.1.0 and deleteing and reconfiguring the service file.
It seems that only 1.1.0 is stable for me for some reason 
When I try to update the db files are being overtaken by the Root user not sure why it happens since I'm using node version 21 so I will wait.
Thank you for the support.
